doc/lib/pgBackRestDoc/Common/DocConfig.pm

```diff
@@ -286,11 +280,14 @@ sub process
 
                     # If a section is specified then use it, otherwise the option should be general since it is not for a specific
                     # command
-                    $strSection = $oOptionDoc->paramGet('section', false);
-
-                    if (!defined($strSection))
+                    if (defined($oOptionDoc))
                     {
-                        $strSection = "general";
+                        $strSection = $oOptionDoc->paramGet('section', false);
+
+                        if (!defined($strSection))
+                        {
+                            $strSection = "general";
```

Comment:
Shouldn't this be inlined without brackets?
```
if (!defined($strSection))
    $strSection = "general";
```

Reply:
I wish. Perl requires the braces even for a single line.

Reply:
Oh right, it's a perl file :man_facepalming:

Reply:
Not too many Perl mods these days, luckily!